extensions/sql-database-projects/src/typings/arm-sql.d.ts

```diff
@@ -0,0 +1,278 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/** ARM resource. */
+export declare interface Resource {
```

Comment:
I assume you're planning on adding this to VS Code MSSQL extension as well? We should definitely do that so others don't have the same issue. 

Although I'd suggest putting all this in the vscode-mssql.d.ts - having to copy a bunch of typings files gets pretty annoying. Just put each of these in their own namespace within the typings file and that should be fine.

Reply:
@Charles-Gagnon I added the types to vscode0mssql.d.ts and I send a separate PR for vscode-mssql. is that the change you were looking for?